src/Parser/Warrior/Fury/Modules/Items/PrePotion.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import ItemLink from 'common/ItemLink';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SUGGESTION_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+
+const debug = false;
+
+const PRE_POTIONS = [
+  SPELLS.POTION_OF_PROLONGED_POWER.id,
+  SPELLS.POTION_OF_THE_OLD_WAR.id,
+];
+
+const SECOND_POTIONS = [
+  SPELLS.POTION_OF_PROLONGED_POWER.id,
+  SPELLS.POTION_OF_THE_OLD_WAR.id,
+];
+
+class PrePotion extends Analyzer {
```

Comment:
I'd prefer not copying this class, but I think the base might not be properly configurable.